### ISSUE TITLE
Add 2nd anchor timer to anchoring notification for structures

### ISSUE THREAD

**Issue by ErikKalkoken** [esi/esi-issues#1190]
# Feature Request

When a structure is anchored there are two timers

- 1st timer starting directly after anchoring
- 2nd timer starting once the anchoring is complete

Anchoring a structure generates a notification of type `StructureAnchoring`.

This notification currently only includes the 1st, but not the 2nd timer. This is very unfortunate, since the 1st timer can be easily derived from the anchoring time (+ 15 min) and the 2nd timer is more complicated. It's 24 hrs for high and null sec, but in null sec it depends on sov parameters and can be up to 7 days. 

I would therefore suggest to add the 2nd timer to the notification or if that is not possible only include the 2nd timer in the notification since it would be way more useful.

Here is how the current notification's text looks like. Both `timeLeft` and `vulnerableTime` refer to the 1st timer:

```yaml
ownerCorpName: My Corporation
solarsystemID: 31000702
structureID: &id001 1010101010101
structureShowInfoData:
- showinfo
- 35832
- *id001
structureTypeID: 35832
timeLeft: 8994617809
vulnerableTime: 9000000000
```

## Use case

Every organization that is using 3rd party tools to monitor their structures.

## Authentication

no

## Example return

I would suggest to add the property `readyTime` to the text, which would represent the LDAP datetime when the anchoring is completed and the 2nd timer for this structure starts.

```yaml
ownerCorpName: My Corporation
solarsystemID: 31000702
structureID: &id001 1010101010101
structureShowInfoData:
- showinfo
- 35832
- *id001
structureTypeID: 35832
timeLeft: 8994617809
vulnerableTime: 9000000000
readyTime: 132307466630000000
```

# Checklist

Check all boxes that apply to this issue:

- [x] Feature request description is provided
- [x] Use case exists
- [ ] Feature requires a new route
- [x] Feature adds data to existing route
- [ ] Feature requires new auth scope
- [x] Feature can reuse existing scope
- [ ] Feature does not require auth
- [ ] Meta feature, applies to all routes

**Comment by lukasni** [comment]
This is out of scope for ESI, the yaml content of the notifications is not defined by ESI but comes directly from the game.
